GoMain/src/main/main.go

```diff
@@ -60,9 +66,13 @@ const (
 	certificateFilePath    = edgeDir + "/data/cert"
 	containerWhiteListPath = edgeDir + "/data/cwl"
 	passPhraseJWTPath      = edgeDir + "/data/jwt"
+	binPath                = "GoMain/bin"
+	YamlFileName           = "/GoMain/bin/res/sample-json-device.yaml"
+	ConfigFileName         = "/GoMain/bin/res/configuration.toml"
 
 	cipherKeyFilePath = edgeDir + "/user/orchestration_userID.txt"
 	deviceIDFilePath  = edgeDir + "/device/orchestration_deviceID.txt"
+	serviceName       = "datastorage"
```

Comment:
I'm a little bit worried that this main file seems to be only for the datastorage service.

Reply:
Incorporated